# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1510176768 18000
#      Wed Nov 08 16:32:48 2017 -0500
# Node ID 51954e5cbbc51c224d0bd485489ed1496286be88
# Parent  2cc557b689d2844bfdb50fbb97a1189c45bedf3a
Fix syntax error introduced in previous merge.

diff --git a/libinterp/parse-tree/oct-parse.in.yy b/libinterp/parse-tree/oct-parse.in.yy
--- a/libinterp/parse-tree/oct-parse.in.yy
+++ b/libinterp/parse-tree/oct-parse.in.yy
@@ -3967,17 +3967,17 @@ namespace octave
          p != lst->end (); p++)
       {
         tree_decl_elt *elt = *p;
 
         std::string nm = elt->name ();
         if (! nm.empty ())
           names.push_back (nm);
       }
-    lexer.mark_as_variables (names);
+    m_lexer.mark_as_variables (names);
 
     switch (tok)
       {
       case GLOBAL:
         {
           retval = new tree_decl_command ("global", lst, l, c);
           retval->mark_global ();
         }
